nixos/tests/buildbot.nix

```diff
@@ -12,6 +12,7 @@ import ./make-test-python.nix {
     bbmaster = { pkgs, ... }: {
       services.buildbot-master = {
         enable = true;
+        pbPort = "'tcp:9989:interface=0.0.0.0'";
```

Comment:
This hides the int bug in the test. The test can only test one of the two possible option types (int or connection string), so why not just leave it as the default?

Reply:
mostly to showcase the more verbose syntax, but right, we have `example` for that.